### DISCUSSION TITLE
Update 0.32.0

### DISCUSSION THREAD

**Discussion by Linkerbrain** [twentyhq/twenty#8276] [Engineering vision]
Hi there!

Big fan of twenty. I saw the sprint of 0.32.0 finished a little while ago, when will this update be released to the hosted instances?

Is there some documentation on the update cycle somewhere?

Best regards,
Linkerbrain

**Comment by BOHEUS** [comment]
It'll be released either in week starting on 4th November or 11th November after core team finishes delivering some critical fixes

**Comment by FelixMalfait** [comment]
Hey, sorry it took some time, it's finally there :). Thanks for your kind words
The team is mostly busy working on workflows/automation which is the major feature coming up (months of work!)!
